### PR TITLE
use current user's avatar as drawer toggle

### PR DESCRIPTION
**Pull request by connyduck**
Screenshot_1598632002

closes https://github.com/tuskyapp/Tusky/issues/1301

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/MainActivity.kt [hunk 1]
```diff
@@ -169,6 +171,8 @@ class MainActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidInje
         val hideTopToolbar = preferences.getBoolean(PrefKeys.HIDE_TOP_TOOLBAR, false)
         mainToolbar.visible(!hideTopToolbar)
 
+        mainToolbar.navigationIcon = FixedSizeDrawable(getDrawable(R.drawable.avatar_default), Utils.dpToPx(this, 36), Utils.dpToPx(this, 36))
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/MainActivity.kt:174]: The navigation icon is not automatically sized correctly, and it ignores bounds set on the drawable, but it works when using this `FixedSizeDrawable` from Glide.

Comment (a1batross) [reply app/src/main/java/com/keylesspalace/tusky/MainActivity.kt:174]: What does 36 mean here?

Looks like magic number to me.

**Comment by a1batross** [comment]
Looks great! :)

File: app/src/main/java/com/keylesspalace/tusky/MainActivity.kt (hunk 1 above)
Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/MainActivity.kt:174]: I'm okay with it being a magic number but we use it 3 times so I would put it into constant.

**Review by charlag — APPROVED** [review]
Works well for me, I like it.
It kinda subverts my expectations of what happens when I click on it but I think it's fine.

**Pull request merged**